packages/fs-detectors/src/monorepos/get-monorepo-default-settings.ts

```diff
@@ -64,15 +64,28 @@ export async function getMonorepoDefaultSettings(
       throw new MissingBuildPipeline();
     }
 
-    return {
+    const rootSettings = {
```

Comment:
Instead of creating multiple variables, what do you think of checking `if (projectPath === '/')` and returning early? In the case when `projectPath === '/'`, then it wouldn't need to evaluate `nonRootSettings`.

Reply:
Agreed 💯